This patch has been submitted upstream, but has not yet been accepted.
http://bugs.python.org/issue22148

--- Python-3.4.1/Python/frozen.c.~1~	2014-03-16 19:31:31.000000000 -0700
+++ Python-3.4.1/Python/frozen.c	2014-08-04 15:06:16.419797757 -0700
@@ -2,7 +2,7 @@
 /* Dummy frozen modules initializer */
 
 #include "Python.h"
-#include "importlib.h"
+#include <importlib.h>
 
 /* In order to test the support for frozen modules, by default we
    define a single frozen module, __hello__.  Loading it will print
